index.d.ts

```diff
@@ -1,13 +1,14 @@
-import type {FileTypeResult} from './core.js';
+import type {FileTypeResult, FileTypeOptions} from './core.js';
 
 /**
 Detect the file type of a file path.
 
 The file type is detected by checking the [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Magic_numbers_in_files) of the buffer.
 
 @param path - The file path to parse.
+@param fileTypeOptions - Optional: An options object including the `customDetectors` property as an Iterable of Detector functions. Those are called in the order provided.
 @returns The detected file type and MIME type or `undefined` when there is no match.
 */
-export function fileTypeFromFile(path: string): Promise<FileTypeResult | undefined>;
+export function fileTypeFromFile(path: string, fileTypeOptions?: FileTypeOptions): Promise<FileTypeResult | undefined>;
```

Comment:
```suggestion
export function fileTypeFromFile(path: string, options?: FileTypeOptions): Promise<FileTypeResult | undefined>;
```

Reply:
Do we really need to expose it on the methods here now that we expose the class? I thought the point of exposing the class was so we didn't have to add the option to each method.

Reply:
Totally agree